### ISSUE TITLE
DevRel Guild

### ISSUE THREAD

**Issue by jlwaugh** [near/devrel#31]
# Open Sourcery ✨

TL;DR ~ this collaboration involving the Community Squad, Open Sourcerers, Berry Pickers, and others would involve biweekly or monthly gatherings of apprentices to connect and learn the magical powers of NEAR development. Over time, individuals may level up, gaining badges / tokens which represent any progress.

## Magic DAOs

This group of councils would be united by the purpose of rewarding developers who help others build meaningful applications on NEAR. Ultimately, our intention is to bring people together and help each other learn to drive progress through creativity. We will construct an engaging narrative around this elite coalition of magical builders, named the Open Sourcerers, a.k.a. NEAR DevRel Guild.

### ACTIONS:

- [ ] Establish "Open Sourcerers" membership structure based on NEAR account IDs, each linked to a forum username
  - [ ] Make typeform used to gather leads for Dev Guilds
  - [ ] Coordinate with AC to integrate with CRM
  - [ ] Explore ideas related to the OpenWeb.Community platform
- [ ] Revamp GitHub repositories
  - [ ] /community
    - [ ] update "quick links" and "join us"
    - [ ] add info re: magical opportunities and link to /devrel
    - [ ] organize hackathon project board
    - [ ] discussions about governance
    - [ ] community bounties!
  - [ ] /devrel
    - [ ] add context in README.md
    - [ ] opensourcery.md
    - [ ] organize hackathons project board
- [ ] Design "Open Sourcery" forum subcategory in the development category as a kind of magic school for getting started ([CYOA example](https://forum.metagame.wtf/t/how-to-cadcad/46))
  - [ ] write category description
  - [ ] initial post explaining how to join and participate
  - [ ] share on various platforms
    - [ ] Twitter
    - [ ] Discord
    - [ ] Telegram
    - [ ] Reddit?
    - [ ] Blog?
- [ ] Organize public meetings (DnD-style campaigns / adventures) where contributors in the NEAR dev community gain knowledge and skills necessary to help others build meaningful applications
  - [ ] Create storylines / narratives / characters based on the Open Web Hack planning document
    - [x] March 5, 12pm EST
    - [ ] March 12, TBD
    - [ ] March 19, TBD
    - [ ] March 26, TBD
  - [ ] Execute promotion strategy
    - [ ] Collaborate on visuals for the Open Sourcery brand and specific adventures
    - [ ] Share announcement with calendar invites on Thursday, February 25
    - [ ] Distribute forum post by Tuesday, March 2
    - [ ] Moar hype on Twitter, Discord, Telegram, and Reddit Wednesday and Thursday, March 3-4
    - [ ] Blog post Friday, March 5 before the actual event?

### TIMELINE:

- February 26: meeting with Open Sourcerers
- March 5, 12, 19, 26: gatherings of NEAR apprentices
- April 16-30: Open Web Hackathon
